examples/gno.land/p/demo/grc/grc20/token.gno

```diff
@@ -250,6 +255,12 @@ func (led *PrivateLedger) Burn(address_XXX std.Address, amount int64) error {
 	return nil
 }
 
+// isKnownAccount checks if the specified address is a known account in the ledger.
+func (led PrivateLedger) isKnownAccount(address_XXX std.Address) bool {
+	_, found := led.balances.Get(address_XXX.String())
+	return found
```

Comment:
```suggestion
func (led PrivateLedger) hasAddr(address_XXX std.Address) bool {
	bal, found := led.balances.Get(address_XXX.String())
	return found && bal > 0
```

Reply:
Why do you want to know if an address with a balance of 0 is known? 

By the way, if I'm not mistaken, we're already freeing up the storage of any address that has a balance of 0, so this check is unnecessary. Please change the name to HasAddr, as we don't want to guarantee the tracking of previous addresses.